lib/webrick-patch.rb

```diff
@@ -0,0 +1,36 @@
+require 'webrick/https'
+
+CIPHERS = 'AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256'
+SSL_VERSION = 'TLSv1_2_server'
```

Comment:
>[weeks@foreman ~]$ irb
>irb(main):001:0> require 'openssl'
>=> true
>irb(main):002:0> OpenSSL::SSL::SSLContext::METHODS
>=> [:TLSv1, :TLSv1_server, :TLSv1_client, :SSLv2, :SSLv2_server, :SSLv2_client, :SSLv3, >:SSLv3_server, :SSLv3_client, :SSLv23, :SSLv23_server, :SSLv23_client]
>irb(main):003:0>

That makes sense actually, I must have missed testing my latest code against el6. I'll update the PR.